backend/e4h-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -989,4 +990,107 @@ async def check_duplicate_tickets(
     finally:
         if input_temp_file and os.path.exists(input_temp_file.name):
-            pass
\ No newline at end of file
+            pass
+
+
+@router.post('/incidents/update',
+             summary='Update incidents from Excel file',
+             response_description='Returns processing results with status for each incident')
+async def update_incidents_from_excel(
+        incidents_file: UploadFile = File(description="Excel file containing incidents to update"),
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)